js1/vstup-vystup-objekty/pripravene.md

```diff
@@ -0,0 +1,33 @@
+## Připravené objekty
+
+Po zkušenosti s objekty jste jistě odhalili, že `document.body.innerHTML` není tak úplně název proměnné, jak jsme se vám snažili do této chvíle namluvit. Z tečkové notace je poznat, že máme co dočinění se zanořenými objekty.
+
+V prohlížeči máme k dispozici celou řadu dopředu připravených objektů, které nám usnadňují práci se stránkou. Jeden takový je uložen v proměnné `document` a poskytuje různé informace o naší stránce.
+
+Například vlastnost `document.title` obsahuje název naší stránky z tagu `<title>`. Snadno jej tak můžeme změnit.
+
+```js
+document.title = 'Objekty v JavaScriptu | Můj blogísek';
```

Comment:
Trochu bych se bál, jestli někoho nenapadne, proč tady se nepoužívá `.innerHTML` a u `document.body` ano. Navrhuji tento příklad vynechat.

Reply:
Pak už ale nezbude nic praktického, co by se na tom `document` dalo použít. Nebo tě něco napadá?

Reply:
Jasný no. Nic moc dobrého mne nenapadá.

Přemýšlel jsem ještě nad `document.location.*`, kdy by se dal dělat třeba redirect, číst, co je za mřížkou, přidávat/číst get parametry, ale bez ifů a tak všeobecně mi to přijde moc komplikované. 🤷‍♀️

Reply:
Aha. Prý by se mělo používat spíš `window.location`.

Reply:
Já navrhuju to tam nechat. Nepřijde mi, že by bylo velké riziko, že někoho zmate to `innerHTML`. Navíc se to po nich nikde v žádném případě neche použít, je to jen ukázka: podívejte, document je složitější objekt, se kterým se dá dělat více věcí. Můžeme se pak zeptat koučů, jestli s tím byl problém a pak to třeba vynechat.

Reply:
Ok